OpenRA.Mods.Common/Traits/Render/RenderDebugState.cs

```diff
@@ -65,7 +65,7 @@ public void OnOwnerChanged(Actor self, Player oldOwner, Player newOwner)
 
 		Color GetColor()
 		{
-			return self.EffectiveOwner != null ? self.EffectiveOwner.Owner.Color.RGB : self.Owner.Color.RGB;
+			return self.EffectiveOwner != null && self.IsInWorld ? self.EffectiveOwner.Owner.Color.RGB : self.Owner.Color.RGB;
```

Comment:
Apparently nothing is null (`t`, `this`, `oldOwner`, `newOwner`) yet it crashes with a `NullReferenceException`:
![image](https://cloud.githubusercontent.com/assets/756669/17085854/22db6c6e-51e3-11e6-8ae7-7afc704d5905.png)

Also handing over `EffectiveOwner` doesn't help. Nor does running the `INotifyOwnerChange after another `AddFrameEndTask`.